### DISCUSSION TITLE
New name for this package: ref-actions?

### DISCUSSION THREAD

**Discussion by denismaier** [emacs-citar/citar#146] [General]
I saw the question on a possible more generic name for this package: https://github.com/bdarcus/bibtex-actions/pull/143#discussion_r650539822

What about ref-actions?

**Comment by bdarcus** [comment]
I kind of think if I go to the trouble of changing the name, it ought to be an even shorter name, and a token; maybe an acronym like embark.

**Reply by denismaier** [reply]
refer would be nice. (But there's already a system with the same name.)

**Reply by denismaier** [reply]
ecamb (emacs citation actions in mini buffer)

**Reply by bdarcus** [reply]
Though with #142, we will also have buffer actions ;-)

**Reply by bdarcus** [reply]
There are also good software names that are more evocative; like ivy and hydra.

**Reply by bdarcus** [reply]
Someone suggested `excite`.

**Reply by andersjohansson** [reply]
Would that be an acronym for Emacs extensive/expressive/excellent/exciting citations? 😁

**Reply by bdarcus** [reply]
I hadn't thought that far ahead :-)

But if it were an acronym, I think the idea of maybe "extensible".
